private/synchronizer/cli/src/main/scala/com/wix/build/sync/cli/ManagedDepsSynchronizerConfig.scala

```diff
@@ -32,6 +33,7 @@ abstract class SynchronizerConfigParser {
     millisBetweenPollings = 3000,
     cacheChecksums = true,
     importExternalLoadStatement = ImportExternalLoadStatement(null, null),
+    failOnMissingArtifacts = true,
```

Comment:
How does default configuration work? If I don't provide anything will it take this value (event if in other classes constructors we are defaulting to false for this parameter)

Reply:
Default value set here is always the value, which will be used executing the command.

I guess your confusion is due to this: https://github.com/wix-playground/rules_maven_third_party/pull/33/files#diff-2168f19f8390ecd73f2c909a958dcd97b1d5630e68c066c28147fbb243794e59R10 - this default value only simplifies unit testing, when creating instances of the class (majority of tests are based on `false`). If this is misleading, let me know.

Reply:
As discussed, changing default value to `false`, to not break current usages.